src/Apps/Order/Components/ExpressCheckout/ExpressCheckoutUI.tsx

```diff
@@ -343,13 +343,16 @@ export const ExpressCheckoutUI = ({ order }: ExpressCheckoutUIProps) => {
               id: orderData.internalID,
               buyerPhoneNumber: phone,
               buyerPhoneNumberCountryCode: null,
-              shippingName: name,
-              shippingAddressLine1: line1,
-              shippingAddressLine2: line2,
-              shippingPostalCode: postal_code,
-              shippingCity: city,
-              shippingRegion: state,
-              shippingCountry: country,
+              shippingName: shippingRate?.id === "PICKUP" ? null : name,
+              shippingAddressLine1:
+                shippingRate?.id === "PICKUP" ? null : line1,
+              shippingAddressLine2:
+                shippingRate?.id === "PICKUP" ? null : line2,
+              shippingPostalCode:
+                shippingRate?.id === "PICKUP" ? null : postal_code,
+              shippingCity: shippingRate?.id === "PICKUP" ? null : city,
+              shippingRegion: shippingRate?.id === "PICKUP" ? null : state,
+              shippingCountry: shippingRate?.id === "PICKUP" ? null : country,
```

Comment:
nitpick: 
```suggestion
              ...(shippingRate?.id === "PICKUP"
                ? {}
                : {
                    shippingName: name,
                    shippingAddressLine1: line1,
                    shippingAddressLine2: line2,
                    shippingPostalCode: postal_code,
                    shippingCity: city,
                    shippingRegion: state,
                    shippingCountry: country,
                  }),
```

Reply:
@MrSltun I believe we need to actually pass the params as null to the backend in order to reset the values. If we skip them from the request they won't update (could be also handled in the backend smartly if the order is pickup but that's not implemented yet)

Reply:
oh in that case then ignore my suggestion 😅